lms/templates/certificates/_accomplishment-rendering.html

```diff
@@ -53,7 +53,7 @@
                 % endif
               </span>
               <h3 class="course-info">
-                ${cert_org_name} | ${course_number} ${_('Certificate')} | ${cert_settings['platform_name_amc']}
+                ${cert_org_name} <span class="string-vertical-divider">|</span> ${course_number} ${_('Certificate')} <span class="string-vertical-divider">|</span> ${cert_settings['platform_name_amc']}
```

Comment:
would you mind putting `${cert_org_name}` into span `<span class="stuff">${cert_org_name}</span>` so customers can customize it further? e.g. add special color to it

Reply:
Done.